src/metrics_server/connection_metrics.ts

```diff
@@ -93,38 +108,70 @@ export function isValidConnectionMetricsReport(
     return false;
   }
 
-  const MIN_BYTES_TRANSFERRED = 0;
-  const MAX_BYTES_TRANSFERRED = 1 * Math.pow(2, 40); // 1 TB.
-  for (const userReport of testObject.userReports) {
-    // Check that `bytesTransferred` is a number between min and max transfer limits
-    if (
-      typeof userReport.bytesTransferred !== 'number' ||
-      userReport.bytesTransferred < MIN_BYTES_TRANSFERRED ||
-      userReport.bytesTransferred > MAX_BYTES_TRANSFERRED
-    ) {
-      return false;
+  let i = testObject.userReports.length;
+  while (i--) {
```

Comment:
The old logic was more readable. The editing of the list makes it harder to understand.
We can just keep the old logic.